autoscale/src/main/java/com/sequenceiq/periscope/service/ScalingActivityService.java

```diff
@@ -42,21 +47,25 @@ public ScalingActivity create(Cluster cluster, ActivityStatus activityStatus, St
         scalingActivity.setActivityStatus(activityStatus);
         scalingActivity.setScalingActivityReason(reason);
         scalingActivity.setStartTime(new Date(creationTimestamp));
+        LOGGER.info("Creating ScalingActivity with creation timestamp: {} for cluster: {}", creationTimestamp, cluster.getStackCrn());
         return scalingActivityRepository.save(scalingActivity);
     }
 
     public void updateWithFlowIdAndTriggerStatus(FlowIdentifier flowIdentifier, String activityCrn, ActivityStatus activityStatus) {
         ScalingActivity scalingActivity = findByCrn(activityCrn);
         scalingActivity.setFlowId(flowIdentifier.getPollableId());
         scalingActivity.setActivityStatus(activityStatus);
+        LOGGER.info("Updating ScalingActivity: {} with FlowInformation and ActivityStatus: {}", activityCrn, activityStatus);
         save(scalingActivity);
     }
 
     public ScalingActivity findByCrn(String activityCrn) {
+        LOGGER.info("Retrieving ScalingActivity by crn: {}", activityCrn);
```

Comment:
Why all the new LOG lines at INFO level? If they're required - sure leave them. Need to be careful about spamming the logs though.

Reply:
Added them to identify points when scalingActivities are being purged. Will limit logging in the service layer to avoid log-spamming